src/Language/PureScript/Ide/Filter/Declaration.hs

```diff
@@ -0,0 +1,57 @@
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE DeriveFoldable  #-}
```

Comment:
Do we really need these extensions? Now that the lens deriving is gone I think we can get rid of these.